### PR TITLE
imporve the export speed

### PR DESCRIPTION
**Pull request by jianwan0214**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Comment by fengttt** [comment]
Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?

File: pkg/frontend/export.go [hunk 1]
```diff
@@ -37,15 +38,16 @@ func (cld *CloseExportData) Close() {
 
 func initExportFileParam(ep *tree.ExportParam, mrs *MysqlResultSet) {
 	ep.DefaultBufSize *= 1024 * 1024
+	fmt.Println("wangjian sql2 is", ep.DefaultBufSize)
```
Comment (daviszhen) [inline pkg/frontend/export.go:41]: remove it

**Comment by jianwan0214** [comment]
> Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?

yes, I changed the string to []byte to avoid some memory copy expenses as convert string to []byte. Here is the ten million rows data test result.

WX20220505-144146@2x

**Comment by jianwan0214** [comment]
> > Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?
> 
> yes, I changed the string to []byte to avoid some memory copy expenses as convert string to []byte. Here is the ten million rows data test result.
> 
> WX20220505-144146@2x

the first is the origin version. the second is the new version.

**Comment by fengttt** [comment]
> > > Can you document what is the change (seems to me, changed from string to []byte), and how much performance improvement you saw?
> > 
> > 
> > yes, I changed the string to []byte to avoid some memory copy expenses as convert string to []byte. Here is the ten million rows data test result.
> > WX20220505-144146@2x
> 
> the first is the origin version. the second is the new version.

You got me really really worried -- is your example saying that, original version take 27 seconds, and the new code take 18 seconds to *CRASH*?

**Comment by jianwan0214** [comment]
> 
no, the second test is caused by restarting the mo-server, not by my code. I need to use two version code, so I need to rebuild the code and restart the mo-server to use the function. I use this just try to compare the two version code. Here is the correct operayion.

**Review by daviszhen — APPROVED** [review]

**Pull request merged**